rs-client-demo/src/main.rs

```diff
@@ -27,133 +27,133 @@ fn main() {
 
     println!("Fetching records using RS client with default Verifier");
 
-    let client = Client::create_with_collection("url-classifier-skip-urls", None);
+    let client = ClientBuilder::new().collection_name("url-classifier-skip-urls").build();
```

Comment:
I kinda like `Client::builder().collection_name("...").build()`

Reply:
yes, I have made the change